### PR TITLE
[Feature]: Group Bar Graph with Recharts

### PR DESCRIPTION
**Pull request by nxanil**
@natabal @killanch 

Added the group bar graph feature https://github.com/nuagenetworks/vis-graphs/issues/223. Please have a look.

grouped-bar-graph

### CODE REVIEW

**Comment by nxanil** [comment]
@killanch 

Added the group bar graph feature #223. Please have a look.

File: Graphs/Components/utils/RechartsTooltip.js
Comment (killanch) [inline Graphs/Components/utils/RechartsTooltip.js:52]: rename Label -> label

File: Graphs/Components/utils/RechartsTooltip.js
Comment (killanch) [inline Graphs/Components/utils/RechartsTooltip.js:57]: why check for `col['payload'][Label]` within conditional? instead, can we have,
```
{Label} : {!isEmpty(col['payload'][Label]) ? columnFormatter(col['payload'][Label]) : col.name}
```

**Review by killanch — COMMENTED** [review]
@nxanil looks good to me, could you pls address minor comments?

**Comment by killanch** [comment]
@preshitg Can you pls review and test these changes? You had earlier worked on https://github.com/nuagenetworks/vis-graphs/pull/292

**Comment by killanch** [comment]
@nxanil Could you pls add unit tests as well?

File: Graphs/Components/utils/RechartsTooltip.js
Comment (nxanil) [reply Graphs/Components/utils/RechartsTooltip.js]: @killanch 

We can not check the above condition with `isEmpty` because we have `string` and `number` types of values. isEmpty is working fine in the case of string but not working with a number.

Comment (killanch) [reply Graphs/Components/utils/RechartsTooltip.js]: @nxanil agreed.
but if `col['payload'][label]` is `0`, it will still not go through columnFormatter since `col['payload'][label]` will resolve as false.
can you pls add a util function in vis-graphs similar to https://github.com/nuagenetworks/vsd-react-ui/blob/master/src/utils/index.js#L1005 and make use of same?

**Comment by preshitg** [comment]
> #292

Verified this similar to #292 and this looks fine.

**Comment by preshitg** [comment]
> @preshitg Can you pls review and test these changes? You had earlier worked on #292

Verified this similar to #292 and this looks fine.

**Comment by nxanil** [comment]
@killanch 

All the feedback changes have been done. unit test cases are also added. Please have a look.

**Review by preshitg — APPROVED** [review]
Changes look good to me. Please also wait for other approvals before merging.

**Review by killanch — APPROVED** [review]

**Review by myTechPlayArea — APPROVED** [review]

**Pull request merged**
